.travis.yml

```diff
@@ -1,6 +1,10 @@
 language: python
 python:
   - '2.7'
+services:
+  - elasticsearch
+before_script:
+  - sleep 10
```

Comment:
@bdero Yes, it's to wait for it to start. It's straight from the [Travis docs](http://docs.travis-ci.com/user/database-setup/).